sbnanalysis/ana/SBNOsc/NumuSelection.h

```diff
@@ -47,12 +58,103 @@ class NumuSelection : public core::SelectionBase {
    */
   bool ProcessEvent(const gallery::Event& ev, std::vector<Event::Interaction>& reco);
 
+  /** Additional information used by the selection per neutrino interaction */
+  struct NuMuInteraction {
+    bool l_is_contained; //!< whether the lepton track is totally contained in the fiducial volume
+    double l_contained_length; //!< the length of the lepton track contained in the fiducial volume
+    double l_length; //!< total length of lepton track
+    double visible_energy; //!< sum of kinetic energies of particles produced directly in interaction
+    double smeared_visible_energy; //!< visible energy with component particle energies smeared
+    double smeared_eccqe; //!< CCQE energy w/ lepton energy smeared
```

Comment:
Please include units in doc strings where relevant.

Reply:
Fixed.